scripts/ImportExportTest.sh

```diff
@@ -0,0 +1,302 @@
+#!/usr/bin/env bash
+
+set -e
+
+# Bash script to test the ast-import option of the compiler by
+# first exporting a .sol file to JSON, then loading it into the compiler
+# and exporting it again. The second JSON should be identical to the first
+READLINK=readlink
+if [[ "$OSTYPE" == "darwin"* ]]; then
+    READLINK=greadlink
+fi
+IMPORT_TEST_TYPE=${1}
```

Comment:
I'd put commandline argument processing as the first thing in the script. Just after `set -e`.